ui/apps/platform/src/Containers/Integrations/IntegrationForm/Forms/ApiTokenIntegrationForm/ApiTokenIntegrationForm.tsx

```diff
@@ -48,6 +51,7 @@ export const validationSchema = yup.object().shape({
 export const defaultValues: ApiTokenIntegrationFormValues = {
     name: '',
     roles: [],
+    expiration: '',
```

Comment:
If this value is required, can we have the default here initialize to what the current default is? (One year?)

Reply:
Also, we should add validation to the `yup` schema above. Without it, the user could leave this field blank and still attempt to submit the form. (Although if a default of one year is provided, it doesn't look like the user will be able to make the form blank.)

At least mark the field as `required`, but it would be nice if validation could cover the correct date format and that the expiration date isn't in the past. `yup` may have helper methods built in to do this easily.